### ISSUE TITLE
[Accessibility]:  More readable legend and total count in restricted viewport

### ISSUE THREAD

**Issue by Numbers88s** [sourcegraph/sourcegraph#35251]
### Audit type

Mobile device navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33506

### Problem description

<img width="284" alt="Screen Shot 2022-05-10 at 12 19 55 PM" src="https://user-images.githubusercontent.com/6225160/167676079-43c072e7-37bd-42f2-bea3-c03dc08d446e.png">

The legend and total count could collapse in a more readable way.

### Expected behavior

We would like to have it be more readable.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by macraig** [comment]
Closing as identified issues have been completed

**Comment by umpox** [comment]
Reopening - I think this should be a quick fix to align them in a column rather than a row, will assign to myself

**Comment by sourcegraph-bot-2** [comment]
Heads up @taylorsperry @jasongornall - the "team/frontend-platform" label was applied to this issue.
